docs/modules/examples/pages/langchain-unstructured-astra.adoc

```diff
@@ -27,7 +27,7 @@ DB Access Token] with Database Administrator permissions.
 Install the following dependencies:
 [source,python]
 ----
-pip install ragstack-ai python-dotenv
+pip install ragstack-ai
```

Comment:
isn't necessary anymore ? I think it is. at line 59 is imported

Reply:
pip install ragstack-ai
pip list
shows python-dotenv 1.0.1
So I figured we don't need it there anymore